dangerfile.js

```diff
@@ -7,3 +7,10 @@ const { danger, fail } = require('danger');
 if (danger.github.pr.body.length === 0) {
   fail('Please include a body for your PR');
 }
+
+if (
+  danger.git.created_files.filter(filename => filename.startsWith('rules/')) &&
+  !danger.git.modified_files.includes('README.md')
```

Comment:
Not sure if this can be improved (unfortunately I don't know and cannot help with that).
1. It only checks for new rules, not for rules that have been renamed for example
2. It doesn't check if the README.md line change was for the specific rule
(but this is still better than nothing, so in my opinion you can leave it like that for now if you want to)

Reply:
Yeah this is far from perfect, but parsing through the actual README text might be error prone & reduce the maintainability of this Danger rule. I opted for the smallest possible solution with the least risk of false failures.

That said - if anyone has specific suggestions to improve this rule, I'm all for it.

Reply:
Good idea! mind splitting this out into a separate PR? 🙂